### DISCUSSION TITLE
[Help] How can I use loader for ui container?

### DISCUSSION THREAD

**Discussion by tonynamy** [fomantic/Fomantic-UI#1446] [Elements (button, image, label, loader, ...)] [answered]
# Help Wanted
I want to use loader for div.ui.container, so I put loader inside the container, but it seems it doesn't overlap all container but only visible area. Since it only overlap the visible area, when I scroll down, the loader doesn't show. 

## Testcase

js fiddle: https://jsfiddle.net/q2nv3Lkz/

![1](https://user-images.githubusercontent.com/36332025/80857164-e5fa8f00-8c8a-11ea-9eef-79f82cbf0e24.gif)

**Comment by ko2in** [comment, marked as answer]
Set `position: relative` to your container. Here's example fiddle (https://jsfiddle.net/y0ucpkqo/1/).

**Comment by YauKeng** [comment]
Facing same problem few days ago, I move the loader to the body, and add css below
```
#loader {
  position: fixed;
  width: 100%;
  height:100%;
  top: 0;
  z-index: 10000;
}
```

example: https://jsfiddle.net/9uathzcb/1/

**Comment by tonynamy** [comment]
Solved! Thanks for your help.
@ko2in, @YauKeng
